src/main/kotlin/lotto/business/PrizeResults.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.business
+
+class PrizeResults(prizeCountMap: Map<LotteryPrize, Int>) {
+
+    private val _prizeCountMap = prizeCountMap.toMutableMap()
+
+    val prizeCountMap: Map<LotteryPrize, Int>
+        get() {
+            return LotteryPrize.values().associateWith {
+                _prizeCountMap.getOrDefault(it, 0)
+            }
+        }
+
+    fun calculateProfitRate(receivedAmount: ReceivedAmount): ProfitRate {
+        val totalPrize = prizeCountMap.map { it.key.prizeAmount * it.value }.sum()
```

Comment:
`prizeCountMap` 대신에 `_prizeCountMap`로 계산하면 0개인 케이스를 제외하고 계산하니 합산을 할 때, 조금 더 유리하지 않을까요? (사실 큰 차이는 없겠지만... 👀)
```suggestion
        val totalPrize = _prizeCountMap.map { it.key.prizeAmount * it.value }.sum()
```